static/src/stylesheets/module/_release-message.scss

```diff
@@ -509,10 +523,15 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 .membership__paypal-container {
     text-decoration: none;
     margin-bottom: $gs-gutter/4;
-    display: flex;
-    margin-top: auto;
+    display: inline-block;
+    margin-top: 5px;
+    margin-bottom: 0px;
     flex-direction: column;
```

Comment:
could you achieve this by changing the `flex-direction` to `row` instead? you might then need a `flex-shrink: 0` on the button to prevent it shrinking (you want it to try and shrink the payment logos to fit instead)

some advantages would be:
- you wouldn't need to mess around with manual widths to try and get it to fit on one line
- you could use `align-items: center` for vertical alignment instead of messing around with manual top/bottom margins